helm/cluster-cloud-director/values.schema.json

```diff
@@ -183,7 +183,23 @@
             "type": "string"
         },
         "kubectlImage": {
-            "type": "object"
+            "type": "object",
+            "title": "kubectl Image",
+            "description": "Used by cluster-shared library chart.",
```

Comment:
Thanks for providing a description! Can we explain _what_ it's being used for?

Reply:
Would [this](https://github.com/giantswarm/cluster-cloud-director/pull/68/commits/f1fee8e223eb9de822dad9af7721a70f0435e03c) be explicit enough?
It is used by [coredns](https://github.com/giantswarm/cluster-shared/blob/9b3382b2737900505f7eee64677db61be7f0a70d/helm/cluster-shared/templates/_coredns.tpl#L279) for now but I suspect it could be used by other components in the future.

Reply:
Totally